connectors/Cloudmersive PDF/apiDefinition.swagger.json

```diff
@@ -0,0 +1 @@
entDate":{"format":"date-time","description":"Date timestamp of the comment","type":"string"},"ReplyChildComments":{"description":"Child comments, that are replies to this one","type":"array","items":{"$ref":"#/definitions/DocxComment"}},"Done":{"description":"True if this comment is marked as Done in Word, otherwise it is false","type":"boolean"}}},"DocxInsertCommentOnParagraphRequest":{"description":"Input to a Insert Comment on Paragraph request","type":"object","properties":{"InputFileBytes":{"format":"byte","description":"Optional: Bytes of the input file to operate on","type":"string"},"InputFileUrl":{"description":"Optional: URL of a file to operate on as input.  This can be a public URL, or you can also use the begin-editing API to upload a document and pass in the secure URL result from that operation as the URL here (this URL is not public).","type":"string"},"ParagraphPath":{"description":"Path to the paragraph to attach the comment to.  You can get the Path by retrieving all of the Paragraphs in document by calling Get Body and taking the Path property of the desired paragraph to add the comment to.","type":"string"},"CommentToInsert":{"$ref":"#/definitions/DocxComment","description":"Comment to insert"}}},"InsertDocxCommentOnParagraphResponse":{"description":"Result of running an Insert-Comment-on-Paragraph command","type":"object","properties":{"Successful":{"description":"True if successful, false otherwise","type":"boolean"},"EditedDocumentURL":{"description":"URL to the edited DOCX file; file is stored in an in-memory cache and will be deleted.  Call Finish-Editing to get the result document contents.","type":"string"}}},"RemoveDocxPagesRequest":{"description":"Input to a Remove Word DOCX Document Pages request","type":"object","properties":{"InputFileBytes":{"format":"byte","description":"Optional: Bytes of the input file to operate on","type":"string"},"InputFileUrl":{"description":"Optional: URL of a file to operate on as input.  This can be a public URL, or you can also use the begin-editing API to upload a document and pass in the secure URL result from that operation as the URL here (this URL is not public).","type":"string"},"StartDeletePageNumber":{"format":"int32","description":"Page number (1-based) to start deleting pages; inclusive","type":"integer"},"EndDeletePageNumber":{"format":"int32","description":"Page number (1-based) to stop deleting pages; inclusive","type":"integer"}}},"GetDocxHeadersAndFootersRequest":{"description":"Input to a Get Tables request","type":"object","properties":{"InputFileBytes":{"format":"byte","description":"Optional: Bytes of the input file to operate on","type":"string"},"InputFileUrl":{"description":"Optional: URL of a file to operate on as input.  This can be a public URL, or you can also use the begin-editing API to upload a document and pass in the secure URL result from that operation as the URL here (this URL is not public).","type":"string"}}},"GetDocxHeadersAndFootersResponse":{"
```

Comment:
@Cloudmersive Could you please fix the indentations as the entire swagger is showing in one line.

Reply:
Fixed

Reply:
@Cloudmersive This swagger looks different and disorganized than the swagger you have submitted in the portal. Please upload the same artifacts here which you have submitted for certification in the portal